client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -28,13 +28,28 @@
     <!-- title column -->
     <ng-container matColumnDef="title">
         <mat-header-cell *matHeaderCellDef mat-sort-header>Topic</mat-header-cell>
-        <mat-cell *matCellDef="let item">{{ item.getListTitle() }}</mat-cell>
+        <mat-cell (click)="onTitleColumn(item)" *matCellDef="let item">{{ item.getListTitle() }}</mat-cell>
```

Comment:
The item number shoud be displayed before the actual title (`<number> · <title>`). Please modify the `getListTitle`.